src/providers/starknet/index.ts

```diff
@@ -59,18 +59,27 @@ export class StarknetProvider extends BaseProvider {
       txs.map(async tx => {
         if (!tx.transaction_hash) return null;
 
-        return this.provider.getTransactionReceipt(tx.transaction_hash);
+        try {
+          return await this.provider.getTransactionReceipt(tx.transaction_hash);
+        } catch (err) {
+          this.log.error(
```

Comment:
Now it's not really an error case, as we kind of expect it to happen and handle it. Maybe it should be `.warn` or `.info` instead?